visualc/VS2010/mbedTLS.vcxproj

```diff
@@ -70,6 +70,7 @@
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|x64'">
     <LinkIncremental>true</LinkIncremental>
     <IntDir>$(Configuration)\$(TargetName)\</IntDir>
+    <OutDir>$(SolutionDir)$(Configuration)\</OutDir>

```

Comment:
For amd64 the previous setting made the lib build to a different directory, than where all of the other projects were looking for, so you couldn't build for that architecture from Visual Studio.